cmd/zc_traverser_local.go

```diff
@@ -37,7 +38,7 @@ type localTraverser struct {
 	fullPath       string
 	recursive      bool
 	followSymlinks bool
-
+	appCtx         context.Context
```

Comment:
nit, let's use ctx for context. It doesn't seem to add any extra value by calling it appCtx.  Is that correct?

Reply:
appCtx to tell its application context, like xdatamove.